mms/model_loader.py

```diff
@@ -137,15 +137,24 @@ def _extract_model(service_name, path):
 
     assert len(glob.glob(os.path.join(model_dir, manifest['Model']['Signature']))) == 1, \
         'Signature file in model archive is inconsistent with manifest.'
+    print("Signature file configured")
 
-    assert len(glob.glob(os.path.join(model_dir, manifest['Model']['Symbol']))) == 1, \
-        'Symbol file in model archive is inconsistent with manifest.'
+    try:
+        if len(glob.glob(os.path.join(model_dir, manifest['Model']['Symbol']))):
+            print("Symbol file configured")
+    except Exception:  # pylint: disable=broad-except
+        print("Symbol file not given. This model service will be treated as an non-symbolic/imperative model service.")
```

Comment:
This can also be the case in case user is using another framework, such as OpenFST. I think a more accurate message would be: "WARNING: Symbol file not provided. This model service will need to handle model configuration".

Reply:
I agree. We should just throw a warning and not talk about model-types.